Source/ProgressionSystemRuntime/Private/Components/PSHUDComponent.cpp

```diff
@@ -168,11 +168,11 @@ void UPSHUDComponent::DisplayLevelUIOverlay(bool IsLevelLocked)
 	if (IsLevelLocked)
 	{
 		// Level is locked show the blocking overlay
-		ProgressionMenuOverlayWidgetInternal->SetOverlayVisibility(ESlateVisibility::Visible);
+		ProgressionMenuOverlayWidgetInternal->SetOverlayVisibility(ESlateVisibility::Visible, true);
```

Comment:
In coding standards, we used to explain to readers what means `true` or false` for secondary bool arguments as others might not understand the context. Please make your bool argument readable
### Expected result
```
constexpr bool bShouldPlayFadeAnimation = true;
ProgressionMenuOverlayWidgetInternal->SetOverlayVisibility(ESlateVisibility::Visible, bShouldPlayFadeAnimation);
```
or
```
ProgressionMenuOverlayWidgetInternal->SetOverlayVisibility(ESlateVisibility::Visible, /*bShouldPlayFadeAnimation*/true);
```